src/pages/components/progress.mdx

```diff
@@ -0,0 +1,84 @@
+---
+title: Progress
+description: >
+  A Progress loader communicates the progress of downloading or uploading content.
```

Comment:
My bad, this sounds a little clunky. Let's try this:

A Progress loader communicates progress when downloading or uploading content.